src/main/webapp/index.jsp

```diff
@@ -105,13 +105,17 @@
                   }
                   %>
               </div>
+              <div id="search-box">
+                <label for="place-input">Search tasks in a different location:</label>
+                <input id="place-input" name="place-input" type="text" placeholder=" Type an address here" class="pac-target-input" autocomplete="off">
```

Comment:
What does `pac` in `pac-target-input` mean in this context?

Reply:
Place Autocomplete, however I just realized I forgot to remove this class attribute here as I am no longer using it :) thanks for pointing this out!